plugins/storage/volume/solidfire/src/main/java/org/apache/cloudstack/storage/datastore/util/SolidFireUtil.java

```diff
@@ -300,7 +311,7 @@ public static String getValue(String keyToMatch, String url, boolean throwExcept
     }
 
     public static String getSolidFireAccountName(String csAccountUuid, long csAccountId) {
-        return "CloudStack_" + csAccountUuid + "_" + csAccountId;
+        return SF_CS_ACCOUNT_PREFIX + csAccountUuid + "_" + csAccountId;
```

Comment:
:) this method could be renamed to compileSolidFireAccountName(..) or formatSolidFireAccountName(..), right? as has been done in recent cleanups in other parts of the code.

Reply:
We could. The idea is really just to get the String representation of the SolidFire account for the CloudStack account in question. I'm definitely open to changing the method name, if you think it would help. The word "compile" is OK, but it typically invokes thoughts of a code compiler to me.

Reply:
What about "create", "generate", "build", "construct", or something similar?

Reply:
So, here's how it's used:

Let's say we need to create a volume.

We know the CloudStack account the volume is associated with. We use this info when calling SolidFireUtil.getSolidFireAccountName to find the name of the SolidFire account that should support this CloudStack account (such a SolidFire account may not exist at this point).

We then use the String that comes back from SolidFireUtil.getSolidFireAccountName and look up the SolidFire account (making an API call to the SolidFire SAN).

If it returns a SolidFire account, we use it. Otherwise, we create a SolidFire account with that name and use it.

Reply:
The method in question (`getSolidFireAccountName`) does not create anything, right? At least I could not see anything there. It basically defines/generates a name that you then use do to some processing (I believe the processing you just described).

Reply:
That sounds a bit better. 

I think that @DaanHoogland mentioned the `get` prefix because we had some other PRs normalizing the methods used to retrieve POJOs attributes. However, if `get` is being used as a synonym for `retrieve` that should also work, which seems to be this case. English is my second language so, I might (probably am) mistaken.

Reply:
Thinking about the `String.format` method, where you send the "pattern" and then the parameters to create a new String. I do believe that the `format` as a prefix looks good.